apisamples/src/main/java/com/eegeo/apisamples/AccuracyRingAroundBlueSphereActivity.java

```diff
@@ -0,0 +1,81 @@
+package com.eegeo.apisamples;
+
+import android.os.Bundle;
+import android.os.Handler;
+
+import com.eegeo.mapapi.EegeoApi;
+import com.eegeo.mapapi.EegeoMap;
+import com.eegeo.mapapi.MapView;
+import com.eegeo.mapapi.geometry.LatLng;
+import com.eegeo.mapapi.map.OnMapReadyCallback;
+import com.eegeo.mapapi.bluesphere.BlueSphere;
+
+public class AccuracyRingAroundBlueSphereActivity extends WrldExampleActivity {
+
+    private MapView m_mapView;
+    private EegeoMap m_eegeoMap = null;
+    private BlueSphere m_bluesphere = null;
+    private boolean m_accuracyRingToggle = false;
+    private Handler m_timerHandler = new Handler();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        EegeoApi.init(this, getString(R.string.eegeo_api_key));
+
+        setContentView(R.layout.bluesphere_change_location_activity);
+        m_mapView = (MapView) findViewById(R.id.bluesphere_change_location_mapview);

```

Comment:
`bluesphere_change_location_mapview` same here. We shouldn't use assets from another example. It can cause this example to break if the other one was changed.